src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -3202,7 +3202,7 @@ def openshift_create(cmd, client, resource_group_name, name,  # pylint: disable=
             workspace_id = '/' + workspace_id
         if workspace_id.endswith('/'):
             workspace_id = workspace_id.rstrip('/')
-        monitor_profile = OpenShiftManagedClusterMonitorProfile(enabled=True, workspace_id=workspace_id)  # pylint: disable=line-too-long
+        monitor_profile = OpenShiftManagedClusterMonitorProfile(enabled=True, workspace_resource_id=workspace_id)  # pylint: disable=line-too-long
```

Comment:
Is this a breaking change? I don't think it's a breaking change.

Reply:
> Is this a breaking change? I don't think it's a breaking change.
> yes, this caused enabling of monitoring failed. Monitor profile should have the enabled=true and workspace_resource_id=<workspace_resource_id> since workspace_id set instead of workspace_resource_id so the Monitor Profile in the request has only enabled flag set not the workspace_resource_id. because of this RP throwing 400 bad request.